platforms/documentation/docs/src/docs/userguide/jvm/java_testing.adoc

```diff
@@ -275,6 +275,18 @@ include::sample[dir="snippets/testing/junit-xml-results/kotlin",files="build.gra
 include::sample[dir="snippets/testing/junit-xml-results/groovy",files="build.gradle[tags=configure-content]"]
 ====
 
+[[junit_xml_configuration_output_filtering]]
+===== includeSystemOutLog & includeSystemErrLog
+
+The `includeSystemOutLog` option allows configuring whether or not test output writen to standard out is exported to the XML report file.
+The `includeSystemErrLog` option allows configuring whether or not test error output written to standard error is exported to the XML report file.
+
+These options affect both test-suite level output (such as `@BeforeClass` output in JUnit 4), and test class and method specific output (`@Before` and `@Test` output in JUnit 4).
+If either option is disabled, the element that would normally contain that content will be omitted from the XML report file.
+
+These options are available when testing with JUnit 4.
```

Comment:
Don't these apply to all testing frameworks?  We generate JUnit xmls for all of them.

Reply:
You're right, even TestNG uses `reports.junitXml` to configure it.  I'll add tests for the other frameworks and expand the scope of the docs and release notes.